app/controllers/users_controller.rb

```diff
@@ -46,6 +46,35 @@ def check_account_creation_status
     end
   end
 
+  # Run a set of corrections on a user
+  def check
```

Comment:
So in unix terms it would be an fsck however I don't think that is useful.

Sanity check, is ableist.

Check user, gives the wrong idea.